pkg/reconciler/v1alpha1/eventlistener/eventlistener.go

```diff
@@ -613,7 +613,7 @@ func (r *Reconciler) reconcileCustomObject(ctx context.Context, logger *zap.Suga
 			}
 		}
 
-		//if dynamicduck.ReconcileCustomObject(existingCustomObject, data) {
+		// if dynamicduck.ReconcileCustomObject(existingCustomObject, data) {
```

Comment:
Should we remove this comment?

Reply:
maybe that's a separate PR...not sure the context behind the comment